static/js/ReaderTextTableOfContents.jsx

```diff
@@ -917,6 +917,9 @@ class ArrayMapNode extends Component {
     if ("refs" in this.props.schema && this.props.schema.refs.length) {
       var sectionLinks = this.props.schema.refs.map(function(ref, i) {
         i += this.props.schema.offset || 0;
+        if (ref == "") {
+          return (null)
```

Comment:
```suggestion
          return null;
```
No need for parentheses. And Javascript lines should end in `;` (although they don't throw an error if they don't have them, it's best practice to include them)